dotcom-rendering/src/types/front.ts

```diff
@@ -400,10 +400,10 @@ export type DCRCollectionType = {
 	 * will always be `false`.
 	 **/
 	canShowMore?: boolean;
-	badge?: DCRBadgeType;
+	editorialBadge?: DCRBadgeType;
+	paidContentBadge?: DCRBadgeType;
+	sponsoredContentBranding?: Branding;
```

Comment:
Tried to be as descriptive as possible with the naming, am very open to suggestions!